NeoProxy/include/NeoProxy/NeoProxy.h

```diff
@@ -99,26 +102,26 @@ struct NEOPROXY_API CDnnBlobDesc {
 // Creates a data blob
 // The blob should be destroyed after use with the help of the DestroyDnnBlob function
 // If an error occurs its description will be written into the errorInfo parameter and the function will return 0
-NEOPROXY_API const struct CDnnBlobDesc* CreateDnnBlob( const struct CDnnMathEngineDesc* mathEngine, TDnnBlobType type,
+NEOPROXY_API const struct CDnnBlobDesc* CreateDnnBlob( const struct CDnnMathEngineDesc* mathEngine, enum TDnnBlobType type,
 	int batchLength, int batchWidth, int height, int width, int depth, int channelCount, struct CDnnErrorInfo* errorInfo );
 
 // Destroys the blob
 NEOPROXY_API void DestroyDnnBlob( const struct CDnnBlobDesc* blob );
 
 // Fills the blob with data from a buffer
-// If successful, returns true; if failed, returns false and fills the errorInfo parameter with the error description
-NEOPROXY_API bool CopyToBlob( const struct CDnnBlobDesc* blob, const void* buffer, struct CDnnErrorInfo* errorInfo );
```

Comment:
what about just including stdbool.h?

Reply:
Vote for bool!
Bool us in C standard more than 20 years